### PR TITLE
BotAPI7.2

### PR DESCRIPTION
**Pull request by Apolisk**

### CODE REVIEW

File: bot.go [hunk 1]
```diff
@@ -1271,6 +1271,27 @@ func (b *Bot) botInfo(language, key string) (*BotInfo, error) {
 	return resp.Result, nil
 }
 
+// GetBusinessConnection use this method to get information about the connection of the bot with a business account.
+// Returns a BusinessConnection object on success.
+func (b *Bot) GetBusinessConnection(businessConnectionID string) (*BusinessConnection, error) {
```
Comment (Nash-Well) [inline bot.go]:
```go
func (b *Bot) BusinessConnection(connectionID string) (*BusinessConnection, error) {
	params := map[string]string{
		"business_connection_id": connectionID,
	}

	data, err := b.Raw("getBusinessConnection", params)
	if err != nil {
		return nil, err
	}

	var resp struct {
		Result *BusinessConnection
	}

	if err := json.Unmarshal(data, &resp); err != nil {
		return nil, wrapError(err)
	}
	return resp.Result, nil
}
```

File: message.go [hunk 2]
```diff
@@ -169,6 +173,16 @@ type Message struct {
...
 
+	// (Optional) Unique identifier of the business connection from which the message
+	// was received. If non-empty, the message belongs to a chat of the corresponding
+	// business account that is independent from any potential bot chat which might
+	// share the same identifier.
+	BusinessConnectionID string `json:"business_connection_id"`
+
+	// (Optional) The bot that actually sent the message on behalf of the business account.
+	// Available only for outgoing messages sent on behalf of the connected business account.
+	SenderBusinessBot *User `json:"sender_business_bot"`
```
Comment (Nash-Well) [inline message.go]:
```go
BusinessBot *User `json:"sender_business_bot"`
```

File: sticker.go [hunk 3]
```diff
@@ -304,3 +303,24 @@ func (b *Bot) SetCustomEmojiStickerSetThumb(name, id string) error {
 	_, err := b.Raw("setCustomEmojiStickerSetThumbnail", params)
 	return err
 }
+
+// ReplaceStickerInSet returns True on success, if existing sticker was replaced with a new one
+func (b *Bot) ReplaceStickerInSet(of Recipient, name, old_sticker string, sticker InputSticker) (bool, error) {
```
Comment (Nash-Well) [inline sticker.go]:
```go
func (b *Bot) ReplaceStickerInSet(of Recipient, name, oldSticker string, sticker InputSticker) (bool, error) {
	files := make(map[string]File)
	
        repr := sticker.File.process("0", files)
	if repr == "" {
		return false, errors.New("telebot: sticker does not exist")
	}
	sticker.Sticker = repr

	data, err := json.Marshal(sticker)
        if err != nil {
             ...
        }

	params := map[string]string{
		"user_id":     of.Recipient(),
		"name":        name,
		"old_stikcer": oldSticker,
		"sticker":     string(data),
	}

	_, err := b.sendFiles("replaceStickerInSet", files, params)
	return true, err
}
```

**Review by Nash-Well — CHANGES_REQUESTED** [review]

File: sticker.go [hunk 4]
```diff
@@ -304,3 +303,29 @@ func (b *Bot) SetCustomEmojiStickerSetThumb(name, id string) error {
...
+
+	data, err := json.Marshal(sticker)
+	if err != nil {
+		return false, err
+	}
+
+	params := map[string]string{
+		"user_id":     of.Recipient(),
+		"name":        name,
+		"old_stikcer": oldSticker,
```
Comment (sofer-eg) [inline sticker.go]: "old_sticker": oldSticker,

Comment (Apolisk) [reply sticker.go]: Thank you, already fixed.

**Pull request merged**